python_profile_finder/ZameenScraper.py

```diff
@@ -2,29 +2,50 @@
 
 from bs4 import BeautifulSoup
 
-from ZameenProfile import ZameenProfile
 from BaseScraper import BaseScraper
 
 
 class ZameenScraper(BaseScraper):
+    base_url = 'https://www.zameen.com'
+    headers_data = {
+        'Referer': 'https://www.zameen.com/login.html',
+        'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64)'
```

Comment:
The value for `User-Agent` is common for all child scrappers. You can either provide this as base class setting or a config variable in base scraper file.